src/index.html

```diff
@@ -7,6 +7,286 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__img">
+        <a href="#">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu is-active"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Samsung </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Smartphones </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+            class="nav__item-menu" data-qa="nav-hover"> Laptops & Computers </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Gadgets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Tablets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Photo </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__item-menu"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+
+
+      <div class="card" data-qa="card">
+        <div class="card__pc">
+          <div class="card__img"></div>
+          <p class="card__descr">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__prodcode">Product code: 195434</p>
+        </div>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+            <div class="stars-star"></div>
+          </div>
+
+          <p class="card__label">Reviews: 5</p>
+        </div>
+
+        <div class="card__buy">
+          <p class="card__price">Price:</p>
+          <p class="card__tag">$2,199</p>
+        </div>
+
+        <a href="#" class="card__button"
+        data-qa="card-hover"> buy </a>
```

Comment:
Fix code style everywhere
```suggestion
        <a
          href="#"
          class="card__button"
          data-qa="card-hover"
        >
          buy
        </a>
```